rsocket-tck-drivers/src/main/java/io/rsocket/tckdrivers/client/JavaClientDriver.java

```diff
@@ -67,6 +84,22 @@ public JavaClientDriver(Supplier<RSocket> createClient) throws FileNotFoundExcep
   }
 
   /**
+   * A function that creates a RSocket on a new TCP connection.
+   *
+   * @return a RSocket
+   */
+  public RSocket createClient(URI uri) {
```

Comment:
You can use the registry to lookup the correct handler by URI

https://github.com/rsocket/rsocket-cli/blob/master/src/main/java/io/rsocket/cli/Main.java#L192